modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-an-operator.adoc

```diff
@@ -13,31 +13,60 @@ link:https://docs.openshift.com/container-platform/latest/operators/olm-what-ope
 
 .Procedure
 
-When deploying {prod-short} using the Operator, configure the intended strategy by modifying the `spec.k8s.ingressStrategy` property of the CheCluster Custom Resource object YAML file.
+When deploying {prod-short} using the Operator, configure the intended strategy by modifying the `spec.server.serverExposureStrategy` and  optionally `spec.server.singleHostExposureType` properties of the CheCluster Custom Resource object YAML file.
 To activate changes done to CheCluster YAML file, do one of the following:
 
 * Create a new cluster by executing the `{prod-cli}` command with applying a patch. For example:
 +
 [subs="+quotes,+attributes"]
 ----
-$ {prod-cli} server:start ... --che-operator-cr-patch-yaml=patch.yaml
+$ {prod-cli} server:start  --installer=operator --platform=... --che-operator-cr-patch-yaml=patch.yaml
 ----
 
-patch.yaml should contain the following:
+patch.yaml should contain the following for `native` single-host exposure type:
 
 [source,yaml,subs="+quotes"]
 ----
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  name: eclipse-che
 spec:
-  k8s:
-    ingressStrategy: '__single-host__'
+  server:
+    serverExposureStrategy: '__single-host__'
+    singleHostExposureType: 'native'
```

Comment:
this property was moved under `k8s` and is available only on raw kube. see https://github.com/eclipse/che-operator/pull/378/files#diff-bc486a26555c78e3a820b89ea45a2d41

Reply:
so on openshift, you cant set `singleHostExposureType` at all and it is always gateway